kfai_sql_chemistry/test/utils/models_for_testing/models.py

```diff
@@ -0,0 +1,25 @@
+from sqlalchemy import Column, ForeignKey, Integer
+from sqlalchemy.orm import declarative_base, relationship
+
+BaseModel = declarative_base()
+
+
+class Address(BaseModel):
+    __tablename__ = 'address'
+
+    id = Column(Integer, primary_key=True)
+
+
+class Person(BaseModel):
+    __tablename__ = 'person'
+
+    id = Column(Integer, primary_key=True)
+    address_id = Column(Integer, ForeignKey('address.id'))
+
+    addresses = relationship(Address)
```

Comment:
If you want a change done to one table to affect both tables in the relationship, add backref.

Also, from the SQLAlchemy documentation, usually the syntax is: 
addresses=relationship("Address")

Reply:
1. Agreed. I don't actually care about the models in this test passed creating a small representation to test DB setup. No bidirectional model relationship will be used/queried. If i happen to use backref here, could i skip having to use the string definition on line 17?

2. I prefer class based references rather than string-based references for change over time. sqlalchemy lets you use either, but i would strongly suggest using the model names directly.

Reply:
1. I don't think so unfortunately :/ You don't have to use backref, I just wanted to bring it up to see what the intention behind the model was 

2. Awesome, I will reflect that in my examples then